review.go

```diff
@@ -14,46 +20,49 @@ const (
 	ReviewClosedReasonRefundedAsFraud ReviewClosedReason = "refunded_as_fraud"
 )
 
-// ReviewOpenedReason describes the reason why the review is opened.
+// The reason the review was opened. One of `rule` or `manual`.
 type ReviewOpenedReason string
 
-// List of values that ReviewOpenedReason can take.
+// List of values that ReviewOpenedReason can take
 const (
 	ReviewOpenedReasonManual ReviewOpenedReason = "manual"
 	ReviewOpenedReasonRule   ReviewOpenedReason = "rule"
 )
 
-// ReviewReasonType describes the reason why the review is open or closed.
-type ReviewReasonType string
+// The reason the review is currently open or closed. One of `rule`, `manual`, `approved`, `refunded`, `refunded_as_fraud`, `disputed`, or `redacted`.
+type ReviewReason string
+
+// Deprecated: we preserve this name for backwards-compatibility, prefer `ReviewReason`.
+type ReviewReasonType = ReviewReason
```

Comment:
is this a breaking change or are type aliasing safe, same for the one below

Reply:
Type aliasing here should be safe.